app/Http/Controllers/Backend/Transport/StationController.php

```diff
@@ -61,4 +63,18 @@ public static function getLatestArrivals(User $user, int $maxCount = 5): Collect
                            ->limit($maxCount)
                            ->get();
     }
+
+    public static function getAlternativeDestinationsForCheckin(TrainCheckin $checkin): Collection {
+        return $checkin->HafasTrip->stopoversNEW
+            ->filter(function(TrainStopover $stopover) use ($checkin) {
+                return $stopover->arrival_planned->isAfter($checkin->departure);
```

Comment:
`arrival_planned` kann null sein (Hier nur Zustieg, oder ausgefallene Station).
```suggestion
                return $stopover->arrival_planned?->isAfter($checkin->departure);
```

Reply:
Nein, kann es nicht mehr seit #1073 

arrival = departure, wenn null